.github/ISSUE_TEMPLATE /BUG_REPORT.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug Report    
+
+about: If you would like to report an issue to seata-ctl, please use this template.
+
+---
+<!-- Please do not use this issue template to report security vulnerabilities but refer to our [security policy](https://github.com/seata/seata/security/policy). -->
+
+- [ ] I have searched the [issues](https://github.com/seata/seata-ctl/issues) of this repository and believe that this is not a duplicate.
+
+### Ⅰ. Issue Description
+
+
+### Ⅱ. Describe what happened
+
+If there is an exception, please attach the exception trace:
+
+```
+Just paste your stack trace here!
+```
+
+
+### Ⅲ. Describe what you expected to happen
+
+
+### Ⅳ. How to reproduce it (as minimally and precisely as possible)
+
+1. xxx
+2. xxx
+3. xxx
+
+Minimal yet complete reproducer code (or URL to code):
+
+
+
+### Ⅴ. Anything else we need to know?
+
+
+### Ⅵ. Environment:
+
+- JDK version(e.g. `java -version`):
```

Comment:
Suggest changing it, it's not a java project.

Reply:
Replaced with 'Go version'